app/views/comments/show.html.erb

```diff
@@ -1,5 +1,8 @@
 <%= flash_div :comment_error, :comment_notice %>
 
-<h3 class="heading"><%=h t('.comment_on', :default => "Comment on") %>  <%= link_to_comment_ultimate_parent(@comment) %></h3>
+<h3 class="heading">
+  <%= t(".comment_on") %>
+  <%= link_to_comment_ultimate_parent(@comment) %>
+</h3>
```

Comment:
From how I understand the Jira issue, this page heading should not be changed. The change you had to add the chapter link in `_single_comment.html.erb` was in the right place.

Reply:
Also, please add a test for this change.